integrations/marketo-v2/lib/index.js

```diff
@@ -4,13 +4,13 @@
  * Module dependencies.
  */
 
-var fmt = require('@segment/fmt');
 var integration = require('@segment/analytics.js-integration');
 var is = require('is');
-var jsonp = require('jsonp');
 var url = require('component-url');
 var when = require('do-when');
 var each = require('@ndhoule/each');
+var onBody = require('on-body-ready');
```

Comment:
I wonder if we simply write our own helper method rather than install a third-party package. The relevant code looks to be about ~10 lines or so. Can we copy/paste?

Reply:
Good idea! I was just following a pattern I saw in another integration but I'd rather not use a dependency. Thanks!

Reply:
Turns out `load` will be enough to make sure the form library is ready to work! we don't need to bring anything in 😎